internal/data/receiver_verification.go

```diff
@@ -75,6 +75,31 @@ func (m ReceiverVerificationModel) GetByReceiverIdsAndVerificationField(ctx cont
 	return receiverVerifications, nil
 }
 
+// GetAllByReceiverId returns all receiver verifications by receiver id.
+func (m ReceiverVerificationModel) GetAllByReceiverId(ctx context.Context, sqlExec db.SQLExecuter, receiverId string) ([]*ReceiverVerification, error) {
+	receiverVerifications := []*ReceiverVerification{}
+	query := `
+		SELECT 
+		    receiver_id, 
+		    verification_field, 
+		    hashed_value,
+		    attempts,
+		    created_at,
+		    updated_at,
+		    confirmed_at,
+		    failed_at
```

Comment:
💡 Since you're returning all the fields, I believe this would be better to maintain:
```suggestion
		    *
```